From b2cdab746ef64bc25ba0b9cff596ebdddd6e8dc6 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 7 Nov 2018 11:30:44 -0800
Subject: [PATCH] Do not include net/ethernet.h and linux/if_ether.h

They conflict when used together especially with musl
removing them still keeps it working so it seems they are redundant

Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
 shared/n-acd/src/n-acd.c              | 1 -
 src/platform/wpan/nm-wpan-utils.h     | 2 --
 src/settings/nm-settings-connection.h | 2 --
 3 files changed, 5 deletions(-)

diff --git a/shared/n-acd/src/n-acd.c b/shared/n-acd/src/n-acd.c
index def56a2..dd4eb78 100644
--- a/shared/n-acd/src/n-acd.c
+++ b/shared/n-acd/src/n-acd.c
@@ -11,7 +11,6 @@
 #include <inttypes.h>
 #include <limits.h>
 #include <linux/if_packet.h>
-#include <netinet/if_ether.h>
 #include <netinet/in.h>
 #include <stdlib.h>
 #include <string.h>
diff --git a/src/platform/wpan/nm-wpan-utils.h b/src/platform/wpan/nm-wpan-utils.h
index 1b54ec4..ed39938 100644
--- a/src/platform/wpan/nm-wpan-utils.h
+++ b/src/platform/wpan/nm-wpan-utils.h
@@ -20,8 +20,6 @@
 #ifndef __WPAN_UTILS_H__
 #define __WPAN_UTILS_H__
 
-#include <net/ethernet.h>
-
 #include "nm-dbus-interface.h"
 #include "platform/nm-netlink.h"
 
diff --git a/src/settings/nm-settings-connection.h b/src/settings/nm-settings-connection.h
index e796b71..c01fef6 100644
--- a/src/settings/nm-settings-connection.h
+++ b/src/settings/nm-settings-connection.h
@@ -22,8 +22,6 @@
 #ifndef __NETWORKMANAGER_SETTINGS_CONNECTION_H__
 #define __NETWORKMANAGER_SETTINGS_CONNECTION_H__
 
-#include <net/ethernet.h>
-
 #include "nm-dbus-object.h"
 #include "nm-connection.h"
 
